### PR TITLE
Remove zfwrite/read_big

### PR DESCRIPTION
**Pull request by zdroid**
Since the functions are equivalent to `fwrite/read_big`, they can be dropped.

### CODE REVIEW

File: src/binio.cpp [hunk 1]
```diff
@@ -14,7 +14,7 @@ size_t fwrite_big(void *buffer,size_t size,size_t count, FILE *stream)
 
 	st=mx/((unsigned long long) size);
 	n=count/st;
-	for(j=0;j<n;j++) 
+	for(j=0;j<n;j++)
```
Comment (matt-gretton-dann) [inline src/binio.cpp:17]: Please drop these stray white-space changes (here and below).

Comment (zdroid) [reply src/binio.cpp:17]: Aren't those white spaces extraneous? My editor automatically strips them (on save). Many editors will do the same. But if you want, I can remove those changes.

Comment (matt-gretton-dann) [reply src/binio.cpp:17]: Please remove these changes.  They are unrelated to the purpose of the commit and so will cause confusion when looking at the log in the long-term.

**Comment by matt-gretton-dann** [comment]
Please also rebase to fix conflicts.

**Comment by zdroid** [comment]
Okay, should be done now.

**Comment by zdroid** [comment]
I dropped a define that's not needed anymore.

**Pull request merged**